metadata/0x4597f91cC06687Bdb74147C80C097A79358Ed29b.json

```diff
@@ -0,0 +1,66 @@
+    "TINLAKE_CURRENCY": "0x6B175474E89094C44Da98b954EedeAC495271d0F",
+    "TITLE": "0xb5E8B7CA8d73F2EEac81F6BAfC5165fba800546c",
+    "PILE": "0x62E6225d9DbFa9C5f09ccB43304F60a0a7dDeb7A",
+    "FEED": "0x479506bFF98b18D62E62862a02A55047Ca6583Fa",
+    "SHELF": "0x35Cbc4F6de5333DAFb372200591E69b68B2A5FbF",
+    "JUNIOR_TRANCHE": "0x377fD64a19190423fa0754767B0fD64C206FdE47",
+    "JUNIOR_TOKEN": "0x2E20940022B4d8aa1E9E63000136D24957324B8B",
+    "JUNIOR_OPERATOR": "0xC4b677A5eD80F32f1f05b593A0b1aA080edC215E",
+    "JUNIOR_MEMBERLIST": "0xE475a3890Df32D30c4b427aAeF37C11c776e81E3",
+    "SENIOR_TRANCHE": "0xb913dd925Fdd34867CBFa492c538B7BdB047F3Cd",
+    "SENIOR_TOKEN": "0x0b304DfFa350B32f608FF3c69f1cE511c11554cF",
+    "SENIOR_OPERATOR": "0x1C8Fb0Ab3694Bc4c0B49402be01ae881ae0D3212",
+    "SENIOR_MEMBERLIST": "0xb4d81aF1E56A7AC07f2ea3cEe7A215deFECF9Bbb",
+    "RESERVE": "0x50f8a1cDEdD9AEf2901C7ec0859D8adbF61B1263",
+    "ASSESSOR": "0x9A359b74BaDd48c23A0A848d149D96B87816F944",
+    "POOL_ADMIN": "0x242B369dee1B298Bb7103F03d0E54974b37Cc1D0",
+    "COORDINATOR": "0x7d0fcC5c1A6B6BdF7f03B28C6207396843db6aBb",
+    "CLERK": "0x58C2fdCa82B7C564777E3547eA13bf8113A015cC",
+    "MAKER_MGR": "0x1F5C294EF3Ff2d2Da30ea9EDAd490C28096C91dF",
+    "MKR_VAT": "0x35D1b3F3D7966A1DFe207aa4514C12a259A0492B",
+    "MKR_JUG": "0x19c0976f590D67707E62397C87829d896Dc0f1F1",
+    "ACTIONS": "0x80F33ED0A69935dd74310b9D0009D0BA647Cf223",
+    "PROXY_REGISTRY": "0xC9045c815bF123ad12EA75b9A7c579C1e05051f9",
+    "AO_REWARD_RECIPIENT": "0xa9253DBe03e86af3a9CEcEf109cbd7A55952EEEe"
+  },
+  "versions": {
+    "FEED": 3,
+    "POOL_ADMIN": 2
+  },
+  "metadata": {
+    "name": "BlockTower Series 1",
+    "shortName": "BT1",
+    "slug": "blocktower-1",
+    "description": "BlockTower Credit is the credit franchise of BlockTower Capital, an SEC-registered digital asset and blockchain investment firm founded in 2017. Within BlockTower Credit, we apply the skills of professional underwriting, structuring and investing at the intersection of real-world credit (a massive but anachronistic traditional industry) and crypto. This Tinlake Pool deploys capital into Whole Loans or Receivables purchased in bulk transactions.",
+    "media": {
+      "logo": "https://storage.googleapis.com/tinlake/pool-media/blocktower-1/logo.svg",
+      "icon": "https://storage.googleapis.com/tinlake/pool-media/blocktower-1/BT1%20Icon.svg",
+      "drop": "https://storage.googleapis.com/tinlake/pool-media/blocktower-1/BT1%20Drop.svg",
+      "tin": "https://storage.googleapis.com/tinlake/pool-media/blocktower-1/BT1%20Tin.svg"
+    },
+    "assetMaturity": "90 days",
+    "attributes": {
+      "Issuer": "BlockTower Series 1",
+      "Links": {
+        "Executive Summary": "https://storage.googleapis.com/tinlake/docs/summaries/BT1.pdf",
+        "Discussion on Discourse": "https://gov.centrifuge.io/tag/blocktower",
+        "Website": "https://www.blocktower.com/"
+      }
+    },
+    "asset": "Whole Loans",
```

Comment:
The configs are missing the Maker config, like in New Silver: https://github.com/centrifuge/tinlake-pools-mainnet/blob/main/metadata/0x53b2d22d07E069a3b132BfeaaD275b10273d381E.json#L57

`ilk`s are `RWA010-A`-`RWA013-A`, `minNonMakerDropShare` should be 0 for all.

Reply:
fixed